Handler.go

```diff
@@ -129,6 +132,32 @@ func events(w http.ResponseWriter, r *http.Request) {
 	}
 }
 
+func setboard(w http.ResponseWriter, r *http.Request){
+
+	var newBoard newBoard
+	var move moveData
+
+	decoder := json.NewDecoder(r.Body)
+	err := decoder.Decode(&newBoard)
+	if err != nil {
+		panic(err)
+	}
+	theGame.board = newBoard.Board
+	move.Turn = theGame.turn
+	hasMove, board := findPotentialMoves(theGame.board, move.Turn)
+	if hasMove {
+		move.Board = board
+	} else {
+		//seems like a sloppy fix
```

Comment:
I agree with this comment....what are you trying to do here.

Reply:
It's supposed to replace the board regardless of the value of hasMove, I was thrown an error when hasMove was assigned a value but never used. I may have misinterpreted the original code though, I assumed that hasMove was supposed to make it remain the original players turn if after they made a move the other player had no valid moves.